react-native-in-app-payments-quickstart/index.js

```diff
@@ -14,8 +14,9 @@
  limitations under the License.
 */
 
-import { AppRegistry } from 'react-native';
+import { AppRegistry, LogBox } from 'react-native';
 import App from './App';
 import { name as appName } from './app.json';
 
+LogBox.ignoreAllLogs();
```

Comment:
Why do we do this?

Reply:
One of our customer raised issue and I resolved this,
https://github.com/square/in-app-payments-react-native-plugin/issues/179

Reply:
Can you add comment to this line explaining why this fix that issue? Is it a workaround to hide the error message or it's a real fix?

Reply:
It's the real solution.